pages/installation/self-hosted/local-docker.mdx

```diff
@@ -0,0 +1,182 @@
+</Callout>
+
+
+Pull in the latest image from docker. Supports both `amd64` and `arm64` CPU architectures.
+
+```shell
+ docker pull mintplexlabs/anythingllm
+```
+
+Mount the storage locally and run AnythingLLM in Docker
+
+<Tabs items={['Linux/Mac', 'Windows']} defaultIndex="0">
+  <Tabs.Tab>
+  ```shell copy showLineNumbers
+  export STORAGE_LOCATION=$HOME/anythingllm && \
+  mkdir -p $STORAGE_LOCATION && \
+  touch "$STORAGE_LOCATION/.env" && \
+  docker run -d -p 3001:3001 \
+  --cap-add SYS_ADMIN \
+  -v ${STORAGE_LOCATION}:/app/server/storage \
+  -v ${STORAGE_LOCATION}/.env:/app/server/.env \
+  -e STORAGE_DIR="/app/server/storage" \
+  mintplexlabs/anythingllm
+  ```
+ </Tabs.Tab>
+
+ <Tabs.Tab>
+  ```powershell copy showLineNumbers
+  $env:STORAGE_LOCATION="$HOME\Documents\anythingllm"; `
+  If(!(Test-Path $env:STORAGE_LOCATION)) {New-Item $env:STORAGE_LOCATION -ItemType Directory}; `
+  If(!(Test-Path "$env:STORAGE_LOCATION\.env")) {New-Item "$env:STORAGE_LOCATION\.env" -ItemType File}; `
+  docker run -d -p 3001:3001 `
+  --cap-add SYS_ADMIN `
+  -v "$env:STORAGE_LOCATION`:/app/server/storage" `
+  -v "$env:STORAGE_LOCATION\.env:/app/server/.env" `
+  -e STORAGE_DIR="/app/server/storage" `
+  mintplexlabs/anythingllm;
+  ```
+ </Tabs.Tab>
+</Tabs>
+
+
+Go to `http://localhost:3001` and you are now using AnythingLLM! All your data and progress will persist between
+container rebuilds or pulls from Docker Hub.
+
+## How to use the user interface
+To access the full application, visit `http://localhost:3001` in your browser.
+
+
+## About UID and GID in the ENV
+- The UID and GID are set to 1000 by default. This is the default user in the Docker container and on most host operating systems. 
+- If there is a mismatch between your host user UID and GID and what is set in the `.env` file, you may experience permission issues.
+
+
+## Build locally from source _not recommended for casual use_
+- `git clone` this repo and `cd anything-llm` to get to the root directory.
+- `touch server/storage/anythingllm.db` to create empty SQLite DB file.
+- `cd docker/`
+- `cp .env.example .env` **you must do this before building**
+- `docker-compose up -d --build` to build the image - this will take a few moments.
+
+Your docker host will show the image as online once the build process is completed. This will build the app to `http://localhost:3001`.
+
+
+## Integrations and one-click setups
+The integrations below are templates or tooling built by the community to make running the docker experience of AnythingLLM easier.
+
+
+### Use the Midori AI Subsystem to Manage AnythingLLM
+
+<Callout type="warning" emoji="️💡">
+  **Note!** ➤➤ Midori AI Subsystem Manager is currently in BETA. If you encounter any issues with the Subsystem Manager, please [contact their team](https://io.midori-ai.xyz/about-us/contact-us/)
+</Callout>
+
+Follow the setup found on [Midori AI Subsystem Tutorial](https://www.youtube.com/watch?v=OPaQ6G5dUIE) for your host OS.
```

Comment:
Same with this one - https://io.midori-ai.xyz/subsystem/manager/ 
Then I can add the video in a "video based how to"